router/core/access_log_field_handler_test.go

```diff
@@ -0,0 +1,125 @@
+package core
+
+import (
+	"errors"
+	"github.com/stretchr/testify/require"
+	"github.com/wundergraph/cosmo/router/internal/expr"
+	"github.com/wundergraph/cosmo/router/internal/requestlogger"
+	"github.com/wundergraph/cosmo/router/pkg/config"
+	"github.com/wundergraph/graphql-go-tools/v2/pkg/operationreport"
+	"go.uber.org/zap"
+	"net/http"
+	"testing"
+)
+
+func TestAccessLogsFieldHandler(t *testing.T) {
+	t.Parallel()
+
+	t.Run("run without any expressions", func(t *testing.T) {
```

Comment:
Do we need to test this separately or can we cover this through an integration test?